source/game/kart/KartCollide.cc

```diff
@@ -399,7 +457,7 @@ void KartCollide::applySomeFloorMoment(f32 down, f32 rate, CollisionGroup *hitbo
 
     projRejSum = projRejSum.rej(nextDir);
 
-    dynamics()->setExtVel(dynamics()->extVel() + projRejSum);
+    dynamics()->setExtVel(dynamics()->extVel() + projRejSum); // projRejSum wrong because for
```

Comment:
```suggestion
    dynamics()->setExtVel(dynamics()->extVel() + projRejSum);
```
Leftover debug comment?